i18n/locales/en.json

```diff
@@ -37,6 +37,6 @@
   "layouts.blogPost.author.byLine": "{author, select, null {} other {By {author}, }}",
   "layouts.blogIndex.currentYear": "News from {year}",
   "components.common.banner.button.text": "Read More",
-  "components.article.author.githubLinkLabel": "{username} Github - opens in new tab",
```

Comment:
Also it looks like you're deleting a translation key from here 🤔

Reply:
Fails from rebase -_-